Framework/PythonInterface/plugins/algorithms/MuonMaxent.py

```diff
@@ -144,6 +141,40 @@ def PyInit(self):
                 optional=PropertyMode.Optional),
             doc="Convergence of phases (optional)")
 
+    def validateInputs(self):
+        issues = dict()
+        first = self.getProperty("FirstGoodTime").value
+        last = self.getProperty("LastGoodTime").value
+
+        if first > last:
```

Comment:
The algorithm also produces an error if the first is equal to last so would it be better to make this greater than or equal to?

Reply:
yes good spot